backend/models/planInvite.ts

```diff
@@ -0,0 +1,65 @@
+import { type PlanInvite } from '@prisma/client'
+import prisma from '../utils/prismaClient'
+const db = prisma
+
+export default class PlanInviteModel {
+  private static getNthDateFromToday(numOfDay: number): Date {
+    const today = new Date()
+    const nthDayFromToday = new Date(today)
+    nthDayFromToday.setDate(nthDayFromToday.getDate() + numOfDay)
+
+    return nthDayFromToday
+  }
+
+  public static async dbCreateOnePlanInvite(planID: string, email: string): Promise<PlanInvite | null> {
+    try {
+      const responseData = await db.planInvite.create({
+        data: {
+          inviteeEmail: email,
+          planID,
+          expiration: PlanInviteModel.getNthDateFromToday(1)
+        }
+      })
+
+      return responseData
+    } catch (error) {
+      console.error(`Error in dbCreateOnePlanInvite: ${error}`)
+      return null
+    }
+  }
```

Comment:
small nit: you can make the `email` arg `inviteeEmail`, which would let you do the short hand object syntax and also be more specific that the email to provide is the `invitee` rather than the `inviter`. Small readability optimization